com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -56,7 +56,7 @@ internal static string PrefabDebugHelper(NetworkPrefab networkPrefab)
         internal SnapshotSystem SnapshotSystem { get; private set; }
         internal NetworkBehaviourUpdater BehaviourUpdater { get; private set; }
 
-        private MessagingSystem m_MessagingSystem;
+        internal MessagingSystem MessagingSystem;
```

Comment:
shouldn't this be a `{ get; private set; }` property?

Reply:
Done.